test/e2e/events.go

```diff
@@ -71,15 +71,15 @@ var _ = Describe("Events", func() {
 		}
 
 		By("submitting the pod to kubernetes")
-		_, err := podClient.Create(pod)
-		if err != nil {
-			Fail(fmt.Sprintf("Failed to create pod: %v", err))
-		}
 		defer func() {
 			By("deleting the pod")
 			defer GinkgoRecover()
```

Comment:
Not really part of your CL, but GinkgoRecover is only necessary in go routines, not defer statements. Please remove this line.